index.js

```diff
@@ -73,7 +73,7 @@ export default class LernaChangelogGeneratorPlugin extends Plugin {
   }
 
   async _execLernaChangelog(from) {
-    let changelog = await this.exec(`${LERNA_PATH} --next-version=${UNRELEASED} --from=${from}`, {
+    let changelog = await this.exec(`node ${LERNA_PATH} --next-version=${UNRELEASED} --from=${from}`, {
```

Comment:
This change seems like a great way to avoid the issue reported! Let's use `process.execPath` though to make sure we use the exact `node` that is being used (this is helpful in certain environments that dynamically determine `node` versions).

```suggestion
    let changelog = await this.exec(`${process.execPath} ${LERNA_PATH} --next-version=${UNRELEASED} --from=${from}`, {
```